lib/screens/import from seed/import_from_seed.dart

```diff
@@ -0,0 +1,288 @@
+                  children: [
+                    Expanded(
+                        child: InputTextField(
+                      controller: _seedPhraseController,
+                      labelText: "Seed Phrase",
+                      maxLines: null,
+                      suffixIcon: IconButton(
+                        onPressed: () {
+                          setState(() {
+                            _seedphrase = !_seedphrase;
+                          });
+                        },
+                        icon: Icon(
+                            color: Colors.white,
+                            _seedphrase
+                                ? Icons.remove_red_eye_outlined
+                                : Icons.visibility_off_outlined),
+                      ),
+                    )),
+                    IconButton(
+                      onPressed: () {},
+                      icon: Image.asset(
+                        'images/qr.png',
+                        width: 60,
+                        height: 60,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              InputTextField(
+                controller: _newPasswordController,
+                validator: newPasswordValidate(),
+                maxLines: 1,
+                labelText: "New Password",
+                obscureText: !_newpassword,
+                helperText: _newpasswordtext,
+                suffixIcon: IconButton(
+                  onPressed: () {
+                    setState(() {
+                      _newpassword = !_newpassword;
+                    });
+                  },
+                  icon: Icon(
+                      color: Colors.white,
+                      _newpassword
+                          ? Icons.remove_red_eye_outlined
+                          : Icons.visibility_off_outlined),
+                ),
+              ),
+              InputTextField(
+                controller: _confirmPasswordController,
+                // validator: confirmPasswordValidate(),
+                obscureText: !_confirmpassword,
+                labelText: "Confirm Password",
+                errorText: (_newPasswordController.text.isNotEmpty &&
+                        _confirmPasswordController.text.isNotEmpty &&
+                        _newPasswordController.text !=
+                            _confirmPasswordController.text)
+                    ? "Your passwords must match. Please try again"
+                    : null,
+                maxLines: 1,
+                suffixIcon: IconButton(
+                  onPressed: () {
+                    setState(() {
+                      _confirmpassword = !_confirmpassword;
+                    });
+                  },
+                  icon: Icon(
+                      color: Colors.white,
+                      _confirmpassword
+                          ? Icons.remove_red_eye_outlined
```

Comment:
Make InputTextField reusable:- avoid to pass in every component the duplicate code like icon, create a separate reusable widget